xrpl/constants.py

```diff
@@ -13,3 +16,7 @@ class XRPLException(Exception):
     """Base Exception for XRPL library."""
 
     pass
+
+
+ISO_CURRENCY_REGEX: Final[Pattern[str]] = compile(f"^[A-Z0-9]{3}$")
+HEX_CURRENCY_REGEX: Final[Pattern[str]] = compile(f"^[A-F0-9]{40}$")
```

Comment:
This will show up in the docs under the "Global Variables" tab - do we want to add docs or remove this from the docs?